.ci/Jenkinsfile

```diff
@@ -143,6 +143,34 @@ pipeline {
         }
       }
     }
+    stage('BDD Tests') {
```

Comment:
Hey @elastic/apm-agent-ruby team. This stage is not necessarily needed, although we see value automating it on CI side.

If so, we should define when it must be run: on PRs, on merges, etc.